src/mlpack/tests/main_tests/hoeffding_tree_test.cpp

```diff
@@ -66,25 +66,25 @@ BOOST_AUTO_TEST_CASE(HoeffdingTreeOutputDimensionTest)
   size_t testSize = testData.n_cols;
 
   // Input training data.
-  SetInputParam("training", std::move(std::make_tuple(info, inputData)));
+  SetInputParam("training", std::make_tuple(info, inputData));
   SetInputParam("labels", std::move(labels));
 
   // Input test data.
-  SetInputParam("test", std::move(std::make_tuple(info, testData)));
+  SetInputParam("test", std::make_tuple(info, testData));
 
   mlpackMain();
 
   // Check that number of output points are equal to number of input points.
   BOOST_REQUIRE_EQUAL(
-      CLI::GetParam<arma::Row<size_t>>("predictions").n_cols, testSize);
+    CLI::GetParam<arma::Row<size_t>>("predictions").n_cols, testSize);
```

Comment:
Ah, these are wrapped lines, so they should be doubly-indented (that is, they were correct before the change).  Can you change them back please?  :+1:

Reply:
> Ah, these are wrapped lines, so they should be doubly-indented (that is, they were correct before the change). Can you change them back please? 👍

Okay, it seems that my IDE automatically formats the code style. Now they changed back.